compiler/testData/codegen/boxInline/complexStack/spillConstructorArgumentsAndInlineLambdaParameter.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_FIR_DIAGNOSTICS
```

Comment:
Variable `res` is again uninitialized, but maybe it was incorrectly initialized in the previous PR.